src/Workspaces/Core/Portable/PatternMatching/PatternMatcher.cs

```diff
@@ -419,9 +416,9 @@ private bool PartStartsWith(string candidate, TextSpan candidatePart, string pat
             => PartStartsWith(candidate, candidatePart, pattern, new TextSpan(0, pattern.Length), compareOptions);
 
         private PatternMatch? TryCamelCaseMatch(
-            string candidate, TextChunk patternChunk,
-            bool punctuationStripped, bool isLowercase,
-            StringBreaks candidateHumps)
+                   string candidate, TextChunk patternChunk,
```

Comment:
:bulb: Previous indentation was correct for several parameter lists changed in this commit.